rmgpy/rmg/input.py

```diff
@@ -252,6 +277,40 @@ def simple_reactor(temperature,
                    constantSpecies=None):
     logging.debug('Found SimpleReactor reaction system')
 
+    try:
+        if mol_to_frag:
+            # calculate total as denominator
+            total = float(0)
+            for initial_mol, value in initialMoleFractions.items():
+                try:
+                    total += value * sum(mol_to_frag[initial_mol].values())
+                except KeyError:
+                    # there might be other species which set cut to be False but in initialMoleFractions
+                    mol_to_frag[initial_mol] = {}
+                    mol_to_frag[initial_mol][initial_mol] = 1
+                    total += value * sum(mol_to_frag[initial_mol].values())
```

Comment:
Minor: for clarity and to reduce some duplication, I think you can refactor this by first checking if `initial_mol` is in `mol_to_frag`. I think something like the following would work, but double check me on this:

```
if initial_mol not in mol_to_frag:  # Likely, this species had cut set to False. Set its count to 1
    mol_to_frag[initial_mol] = {initial_mol: 1}  # The whole species is the only fragment

total += value * sum(mol_to_frag[initial_mol].values())


```